src/frontend/src/catalog/system_catalog/rw_catalog/rw_types.rs

```diff
@@ -24,19 +24,34 @@ use risingwave_common::types::{DataType, ScalarImpl};
 use crate::catalog::system_catalog::{BuiltinTable, SysCatalogReaderImpl};
 
 macro_rules! impl_pg_type_data {
-    ($( { $enum:ident | $oid:literal | $oid_array:literal | $name:ident | $input:ident | $len:literal } )*) => {
+    ($( { $enum:ident | $oid:literal | $oid_array:literal | $name:ident | $input:ident | $len:literal | $elem:literal} )*) => {
         &[
             $(
-            ($oid, stringify!($name), stringify!($input)),
+            ($oid, stringify!($name), stringify!($input),$elem),
             )*
             // Note: rw doesn't support `text` type, returning it is just a workaround to be compatible
             // with PostgreSQL.
-            (25, "text", "textin"),
-            (1301, "rw_int256", "rw_int256_in"),
+            (25, "text", "textin",0),
+            (1301, "rw_int256", "rw_int256_in",0),
+            // Note: Here is only to avoid some components of psql from not being able to find relevant results, causing errors. We will not use it in the RW.
+            (1000, "_bool", "array_in",16),
```

Comment:
The existing macro already has enough info to produce entries for both element and array:
```
    $(
    ($oid, stringify!($name), stringify!($input), 0, $oid_array),
    )*
    $(
    ($oid_array, concat!("_", stringify!($name)), "array_in", $oid, 0),
    )*
```
so that we will not have the bug of forgetting `_int4`.